frontend/src/app/components/action-plan/action-plan.component.html

```diff
@@ -1,23 +1,17 @@
 <div class="okr-form-row okr-form-label-input-container">
-  <label for="actionlist" class="okr-form-label">Action Plan (optional)</label>
+  <label for="action-list" class="okr-form-label">Action Plan (optional)</label>
   <div
     cdkDropList
     *ngIf="control?.getValue()"
     [cdkDropListData]="control | async"
     (keydown.enter)="preventAddingNewItems($event)"
     class="okr-form-col d-flex flex-column gap-2"
     (cdkDropListDropped)="drop($event)"
-    id="actionlist"
+    id="action-list"
   >
     <div class="action-point-item" *ngFor="let action of (control | async)!; index as i; trackBy: trackByFn" cdkDrag>
       <div class="drag-drop-dots">
-        <img
-          cdkDragHandle
-          class="icons"
-          src="../../../assets/icons/drag_indicator.svg"
-          alt="Drag indicator"
-          [attr.data-testId]="'dragIcon'"
-        />
+        <img cdkDragHandle class="icons" src="../../../assets/icons/drag_indicator.svg" alt="Drag indicator" />
```

Comment:
why is the test-id removed?

Reply:
i guess it got removed because it had 0 ussages